// @wdio/types/build/Workers.js
/**
 * The following defines a new worker messaging system
 */
export var MESSAGE_TYPES;
(function (MESSAGE_TYPES) {
    /**
     * @wdio/browser-runner messages
     */
    MESSAGE_TYPES[MESSAGE_TYPES["consoleMessage"] = 0] = "consoleMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["commandRequestMessage"] = 1] = "commandRequestMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["commandResponseMessage"] = 2] = "commandResponseMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["hookTriggerMessage"] = 3] = "hookTriggerMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["hookResultMessage"] = 4] = "hookResultMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["expectRequestMessage"] = 5] = "expectRequestMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["expectResponseMessage"] = 6] = "expectResponseMessage";
    MESSAGE_TYPES[MESSAGE_TYPES["coverageMap"] = 7] = "coverageMap";
    MESSAGE_TYPES[MESSAGE_TYPES["customCommand"] = 8] = "customCommand";
    MESSAGE_TYPES[MESSAGE_TYPES["initiateBrowserStateRequest"] = 9] = "initiateBrowserStateRequest";
    MESSAGE_TYPES[MESSAGE_TYPES["initiateBrowserStateResponse"] = 10] = "initiateBrowserStateResponse";
    MESSAGE_TYPES[MESSAGE_TYPES["browserTestResult"] = 11] = "browserTestResult";
    /**
     * @wdio/runner messages
     * TODO: add runner messages
     */
})(MESSAGE_TYPES || (MESSAGE_TYPES = {}));
